docs-chef-io/content/inspec/resources/group.md

```diff
@@ -15,91 +15,109 @@ Use the `group` Chef InSpec audit resource to test a single group on the system.
 
 The `group` resource uses the following system groups:
 
-- On **non-Windows** systems the group resource tests a local group defined in the`/etc/group` file.
-
-- On **Windows** systems the group resource tests a local group defined by Local Users and Groups.
+- On **non-Windows** systems, the group resource tests a local group defined in the`/etc/group` file.
 
+- On **Windows** systems, the group resource tests a local group defined by Local Users and Groups.
 
 ## Availability
 
 ### Installation
 
-This resource is distributed along with Chef InSpec itself. You can use it automatically.
+The Chef InSpec resource distributes this resource.
 
 ### Version
 
-This resource first became available in v1.0.0 of InSpec.
+This resource is available from Chef Inspec 1.0.0.
 
 ## Syntax
 
-A `group` resource block declares a group, and then the details to be tested, such as if the group is a local group, the group identifier, or if the group exists:
+A `group` resource block declares a group and the details to be tested, such as if the group is a local group, the group identifier, or if the group exists.
 
+```ruby
     describe group('group_name') do
       it { should exist }
       its('gid') { should eq 0 }
     end
+```
 
-where
-
-- `'group_name'` must specify the name of a group to be tested on the system
-- `exist` and `'gid'` are valid matchers for this resource
+> where
+>
+> - `group_name` must specify the group's name to be tested on the system.
+> - `exist` and `gid` are valid matchers for this resource.
 
 ## Properties
 
 ### gid
 
-The `gid` property returns the named group identifier:
+The `gid` property returns the named group identifier.
 
+```ruby
     its('gid') { should eq 1234 }
+```
 
 ### members
 
-The `members` property returns the members that belong to the group:
+The `members` property returns the members that belong to the group.
 
+```ruby
     its('members') { should include 'root' }
-
-where `members` returns:
-
-- an array of group members for **Windows Platform**.
-
-    Example: `["member1", "member2"]`
-
-- a CSV formatted string of group members for **Non-Windows Platforms**.
-
-    Example: `"member1,member2"`
+```
+
+> where `members` returns:
```

Comment:
Just one doubt, if the following indentation is intentional. The example lines seems to be misaligned. If it is intentional, not a problem.
<img width="738" alt="image" src="https://user-images.githubusercontent.com/98935583/163816169-c3f689b2-5865-493c-b0be-50d61d46919d.png">